connector/rest/src/main/java/it/pagopa/selfcare/external_api/connector/rest/MsPartyRegistryProxyConnectorImpl.java

```diff
@@ -32,4 +43,12 @@ public InstitutionResource findInstitution(String institutionExternalId) {
         return result;
     }
 
+    @Override
+    public LegalVerification verifyLegal(String taxId, String vatNumber){
+        log.trace("verifyLegal start");
```

Comment:
you could add some referente to taxId and vatNumber....to avoid log injection you can use Encode.forJava